lib/DDG/Spice/Nutrition.pm

```diff
@@ -24,7 +39,12 @@ handle query_lc => sub {
     if (/^(.+?)\s?$question_regex$attribute_regex(?:\?)?/) {
         return $1 if $1;
     }
-
+    
+    # check for matching fruits, vegetables, animal products, meat, ...
+    if ( grep( /^$_$/, @food_triggers ) ) {
```

Comment:
I give it a try during this week. In my opinion we should implement handling plurals on top of this #3262 PR (after review)?